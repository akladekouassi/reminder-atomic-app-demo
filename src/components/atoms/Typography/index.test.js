import { render } from "enzyme";

import Typography from ".";

describe("Typography", () => {
  it("Renders Default Body", () => {
    const wrapper = render(<Typography>Body</Typography>);
    expect(wrapper).toBeDefined();
    expect(wrapper).toMatchSnapshot();
  });
  it("Renders Body", () => {
    const wrapper = render(<Typography variant="body">Body</Typography>);
    expect(wrapper).toBeDefined();
    expect(wrapper).toMatchSnapshot();
  });
  it("Renders Header", () => {
    const wrapper = render(<Typography variant="header">Header</Typography>);
    expect(wrapper).toBeDefined();
    expect(wrapper).toMatchSnapshot();
  });
  it("Renders Title", () => {
    const wrapper = render(<Typography variant="title">Title</Typography>);
    expect(wrapper).toBeDefined();
    expect(wrapper).toMatchSnapshot();
  });
  it("Renders Subtitle", () => {
    const wrapper = render(
      <Typography variant="subtitle">Subtitle</Typography>
    );
    expect(wrapper).toBeDefined();
    expect(wrapper).toMatchSnapshot();
  });
});
