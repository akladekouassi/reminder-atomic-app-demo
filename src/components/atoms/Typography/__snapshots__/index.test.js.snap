// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typography Renders Body 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "MuiTypography-root MuiTypography-body1",
    },
    "children": Array [
      Object {
        "data": "Body",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "text",
      },
    ],
    "name": "p",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;

exports[`Typography Renders Default Body 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "MuiTypography-root MuiTypography-body1",
    },
    "children": Array [
      Object {
        "data": "Body",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "text",
      },
    ],
    "name": "p",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;

exports[`Typography Renders Header 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "MuiTypography-root MuiTypography-h4",
    },
    "children": Array [
      Object {
        "data": "Header",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "text",
      },
    ],
    "name": "h4",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;

exports[`Typography Renders Subtitle 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "MuiTypography-root MuiTypography-subtitle2",
    },
    "children": Array [
      Object {
        "data": "Subtitle",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "text",
      },
    ],
    "name": "h6",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;

exports[`Typography Renders Title 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "MuiTypography-root MuiTypography-subtitle1",
    },
    "children": Array [
      Object {
        "data": "Title",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "text",
      },
    ],
    "name": "h6",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
